src/NuGet.Core/NuGet.Build.Tasks/NuGet.props

```diff
@@ -10,22 +10,32 @@ Copyright (c) .NET Foundation. All rights reserved.
 ***********************************************************************************************
 -->
 
-<Project>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
The NETFX MSBuild will complain if there is no namespace. Can you confirm, whether this gets imported in NETFX MSBuild?

Reply:
It depends on what exactly you mean by "netfx".

If you mean the msbuild.exe that ships with Visual Studio 2019, yes the props file gets imported, but if it was a problem, then msbuild would have been complaining since this file was first inserted into VS or the .NET SDK. Also [msbuild's docs on the project element](https://docs.microsoft.com/en-us/visualstudio/msbuild/project-element-msbuild?view=vs-2017) for both VS2017 and VS2019 explicitly say that the xmlns is optional. If it wasn't, then msbuild.exe wouldn't support SDK style projects, since they don't define the xmlns either.

If by netfx you mean the .NET Framework Targeting Pack, then no, nuget.targets doesn't ship there. The msbuild that ships there, even with .NET 4.8, is super old, something like MSBuild 8 from memory. I'm sure MSBuild skipped some numbers, but NuGet's PackageReference was first supported in msbuild 15.x.

Reply:
That's why, I was asking whether NETFX MSBuild will import the common props (from the installed version that has NuGet props). If it doesn't then we don't need xmlns. If that's the case then I could remove them from all the files.

Reply:
If that's the case then we could remove `xmlns` from restore and pack targets. Is that okay with you?